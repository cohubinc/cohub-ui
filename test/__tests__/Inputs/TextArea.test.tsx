import React from "react";
import { mount } from "enzyme";

import { Inputs } from "dist";

describe("TextArea Input", () => {
  it("renders without crashing", async () => {
    const wrapper = mount(<Inputs.TextArea />);

    expect(wrapper).toMatchSnapshot();
  });
});
