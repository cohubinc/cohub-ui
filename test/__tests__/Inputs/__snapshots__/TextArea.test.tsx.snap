// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea Input renders without crashing 1`] = `
<TextArea
  className=""
  style={
    Object {
      "cursor": "text",
      "width": "100%",
    }
  }
>
  <textarea
    className="GenericTextArea border bd-radius "
    style={
      Object {
        "cursor": "text",
        "outline": "none",
        "padding": "10px 12px",
        "width": "100%",
      }
    }
  />
</TextArea>
`;
